src/templates/app/user/security/self_exclusion.jsx

```diff
@@ -82,23 +82,18 @@ const SelfExclusion = () => (
 
                 <FormRow type='text' id='exclude_until' label={it.L('Exclude me from the website until')} attributes={{ maxLength: 15, autoComplete: 'off', readOnly: 'readonly' }} className='clearable' hint={it.L('Please enter date in the format DD MMM, YYYY.')} />
 
-                <div id='gamstop_info_bottom' className='gr-row invisible'>
-                    <div className='gr-8 gr-push-4 gr-12-m gr-push-0-m'>
-                        <p className='notice-msg'>{it.L('To self-exclude from all online gambling companies licensed in Great Britain, go to [_1].', '<a target="_blank" rel="noopener noreferrer" href="https://www.gamstop.co.uk">www.gamstop.co.uk</a>')}</p>
-                    </div>
-                </div>
-
-                <SubmitButton text={it.L('Update settings')} type='submit' />
-
-                <div id='ukgc_requirement_notice' data-show='iom, malta' className='gr-row invisible'>
+                <div id='self_exclusion_warning' data-show='iom, malta' className='gr-row gr-padding-10 invisible'>
                     <div className='gr-8 gr-push-4 gr-12-m gr-push-0-m'>
                         <div className='notice-msg gr-child gr-parent'>
                             <p>{it.L('Self-exclusion on this website only applies to your [_1] account and does not include other companies or websites.', it.website_name)}</p>
-                            <p>{it.L('Since you feel the need to self-exclude, we strongly encourage you to exclude yourself from similar services that you may have signed up for.')}</p>
-                            <p id='ukgc_gamstop'>{it.L('To self-exclude from all online gambling companies licensed in Great Britain, you may go to [_1].', '<a target="_blank" rel="noopener noreferrer" href="https://www.gamstop.co.uk">www.gamstop.co.uk</a>')}</p>
+                            <p><span id='ukgc_gamstop' className='invisible' >{it.L('If you are a UK resident, ')}</span>{it.L('To self-exclude from all online gambling companies licensed in Great Britain, you may go to [_1].', '<a target="_blank" rel="noopener noreferrer" href="https://www.gamstop.co.uk">www.gamstop.co.uk</a>')}</p>
```

Comment:
please use placeholders before and after `[_1]If you are a UK resident, [_2] To self-exclude from all...` and put the span in the placeholders so translators can translate the full sentence to restructure it for grammar etc, and instead of `ukgc_gamstop` use `data-show='malta'` if possible